ui/src/components/Organization/List.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames/bind';
+import style from 'src/components/Organization/style.module.scss';
+
+const c = classNames.bind(style);
+
+interface Organizations {
+  id: string;
+  name: string;
+  banner?: string;
+  square_logo?: string;
+}
+
+interface FieldProps {
```

Comment:
This name seem copied from some Field component. Please beware when copy-pasting code. Even an agnostic `Props` would work, but `FieldProps` is misleading.